ts/features/wallet/cobadge/component/CobadgeWalletPreview.tsx

```diff
@@ -51,6 +52,15 @@ const renderLeft = (props: Props, size: Option<[number, number]>) =>
     }
   );
 
+const getAccessibilityRepresentation = (cobadge: CreditCardPaymentMethod) => {
+  const cardRepresentation = I18n.t("wallet.accessibility.folded.coBadge", {
+    brand: cobadge.info.brand,
+    bankName: cobadge.abiInfo?.name
```

Comment:
It could be `undefined` and when it is this happens

https://user-images.githubusercontent.com/822471/128691704-5194b821-18ba-42e4-b1bf-9336ad488e54.MP4

```suggestion
    bankName: cobadge.abiInfo?.name ?? "not available [with I18n]"
```

Reply:
Opsss 😅, changed in ef297af